tests/standalone_tests/conftest.py

```diff
@@ -90,26 +92,35 @@
 CONTROLLER_PORT = 4500
 CONTROLLER_SMPC_PORT = 4501
 
-GLOBALNODE_CONFIG_FILE = "testglobalnode.toml"
-LOCALNODE1_CONFIG_FILE = "testlocalnode1.toml"
-LOCALNODE2_CONFIG_FILE = "testlocalnode2.toml"
-LOCALNODETMP_CONFIG_FILE = "testlocalnodetmp.toml"
-GLOBALNODE_SMPC_CONFIG_FILE = "smpc_globalnode.toml"
-LOCALNODE1_SMPC_CONFIG_FILE = "smpc_localnode1.toml"
-LOCALNODE2_SMPC_CONFIG_FILE = "smpc_localnode2.toml"
-CONTROLLER_CONFIG_FILE = "testcontroller.toml"
-CONTROLLER_SMPC_CONFIG_FILE = "test_smpc_controller.toml"
-CONTROLLER_LOCALNODES_CONFIG_FILE = "test_localnodes_addresses.json"
-CONTROLLER_SMPC_LOCALNODES_CONFIG_FILE = "test_smpc_localnodes_addresses.json"
+GLOBALNODE_CONFIG_FILE = "globalnode.toml"
+LOCALNODE1_CONFIG_FILE = "localnode1.toml"
+LOCALNODE2_CONFIG_FILE = "localnode2.toml"
+LOCALNODETMP_CONFIG_FILE = "localnodetmp.toml"
```

Comment:
having the "test" prefix on those nodes was making it much easier to discriminate between the nodes/containers/wahteverothercomponent of the deployment vs the standalone tests. Why would you remove it?

Reply:
These names are for the config files not for container names. Since all these files are in the test folder, is there a reason to prefix?

Reply:
Having the exact same names for the config files, container names or any other name that refers to these nodes would make things easier to follow.

Reply:
fixed